### ISSUE TITLE
Jumpy Christine

### ISSUE THREAD

**Issue by FabianNitsche** [csinkers/ualbion#90]
There are two instances of Christine at the very start of the game (Toronto). One on the bed and one at the door.

![grafik](https://github.com/user-attachments/assets/728000b6-e35b-4c48-9d50-fcf4882a7198)

The one on the bed is gone after some time.
When going out my room, Christine does not approach me.

![grafik](https://github.com/user-attachments/assets/f0ce4290-8b05-4629-ad75-b265a2ad0fef)

The Christine from the bed seems to jump somewhere else right before that.

**Comment by FabianNitsche** [comment]
Mabe this is something I can investigate further.

**Comment by FabianNitsche** [comment]
I found her in the first cargo bay.
![grafik](https://github.com/user-attachments/assets/ddc1f572-7309-4dbf-a0ab-a691f3c96caa)

And the conversation window is blank.

I had this issue before. It seems related to asynchronous tasks not completing, because this also went away when I went back in commit history (issue #89).

So maybe even the strange behavior of Christine is related.

**Comment by FabianNitsche** [comment]
If I end the conversation also the conversation window is stuck:
![grafik](https://github.com/user-attachments/assets/f7180773-e5fb-4d4a-9916-764838e71468)

**Comment by csinkers** [comment]
Yeah, conversations have been broken for a while. Working on fixing them up at the moment.

**Comment by csinkers** [comment]
Also - the second Christine probably isn't actually a bug: consider that in the original game the view is zoomed in far enough that you can't see into her room. Later on I'm planning to put together an optional 'data fixes' mod that will fix various things in the original game's data files, updating the MapInit script for this map to hide Christine could be a part of that.

**Comment by csinkers** [comment]
Christine not approaching you during the cutscene when you leave the room is definitely a ualbion bug though. It'll be related to the exact timing of 'update' events and the speed that NPCs move, that's definitely an area that needs some fine tuning.
